### PR TITLE
Make "false" mean false instead of true for maps feature flag

### PR DESCRIPTION
**Pull request by timabell**
Gotcha!

### CODE REVIEW

File: shared/Features/FeatureFlags.cs [hunk 1]
```diff
@@ -8,13 +8,17 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.Features
     public class FeatureFlags : IFeatureFlags
     {
         private readonly IConfiguration _config;
-        private const string APPLY_2019 = "FEATURE_APPLY_2019";
+        private const string FeaturePrefix = "FEATURE_";
```
Comment (dankmitchell) [inline shared/Features/FeatureFlags.cs]: This is overkill IMO 🤷‍♂️

Comment (timabell) [reply shared/Features/FeatureFlags.cs]: Old habits DRY hard

File: shared/Features/FeatureFlags.cs [hunk 2]
```diff
@@ -8,13 +8,17 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.Features
...
+        private const string Apply2019Key = FeaturePrefix + "APPLY_2019";
+        private const string MapsKey = FeaturePrefix + "MAPS";
 
         public FeatureFlags(IConfiguration config)
         {
             _config = config;
         }
-        public bool Apply2019 => ShouldShow(APPLY_2019);
+
+        public bool Apply2019 => ShouldShow(Apply2019Key);
```
Comment (benilovj) [inline shared/Features/FeatureFlags.cs]: This can go now 😆

Comment (benilovj) [reply shared/Features/FeatureFlags.cs]: we can remove this in a subsequent PR

File: shared/Features/FeatureFlags.cs [hunk 3]
```diff
@@ -8,13 +8,14 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.Features
...
         private readonly IConfiguration _config;
-        private const string APPLY_2019 = "FEATURE_APPLY_2019";
 
         public FeatureFlags(IConfiguration config)
         {
             _config = config;
         }
-        public bool Apply2019 => ShouldShow(APPLY_2019);
+
+        public bool Apply2019 => ShouldShow("FEATURE_APPLY_2019");
```
Comment (dankmitchell) [inline shared/Features/FeatureFlags.cs]: Delete this?

File: shared/Features/FeatureFlags.cs (hunk 2 above)
Comment (timabell) [reply shared/Features/FeatureFlags.cs]: https://trello.com/c/4XxSadm2/475-remove-apply-feature-flag

File: shared/Features/FeatureFlags.cs (hunk 3 above)
Comment (timabell) [reply shared/Features/FeatureFlags.cs]: https://trello.com/c/4XxSadm2/475-remove-apply-feature-flag

Comment (timabell) [reply shared/Features/FeatureFlags.cs]: I'm trying to keep each PR tight in scope, they have a habit of getting out of hand!

**Pull request merged**